gradle.properties

```diff
@@ -2,15 +2,15 @@ kotlin.code.style=official
 org.gradle.jvmargs=-Xmx2G
 
 modId=fabric-language-kotlin
-modVersion=1.8.2
+modVersion=1.8.3
 minecraftVersion=1.19
-mappingsVersion=1.19+build.1
-loaderVersion=0.14.7
+mappingsVersion=1.19.2+build.3
```

Comment:
revert this since you didn't update everything to 1.19.2

Reply:
fixed